### DISCUSSION TITLE
How to change address in share link from LOCALHOST to public one

### DISCUSSION THREAD

**Discussion by vladobb** [photoprism/photoprism#4671] [Q&A] [answered]
hi friends, I am running PP on unraid docker, with working SWAG, i.e. https://fotky.buzalka.cz is working well. But when i click on create share link I am proposed with link in format http://localhost:2342/s/obpnsfby6f/test but i need https://fotky.buzalka.cz/s/obpnsfby6f/test

Many thanks for your advice

**Comment by omerdduran** [comment, marked as answer]
Hi. First, make sure you configure the correct site URL in your compose file by setting the PHOTOPRISM_SITE_URL environment variable to your public domain. 

```yaml
environment:
  PHOTOPRISM_SITE_URL: "https://your-domain.com/"  # Replace with your actual domain
```

For security reasons, when exposing PhotoPrism to the internet, you should:

- Always use HTTPS by putting PhotoPrism behind a secure reverse proxy

After making these changes:
- Restart your PhotoPrism containers

**Reply by vladobb** [reply]
Many thanks, one would expect that docker author expose this variable in docker setup :-), there is title, caption, author, but not site url. It works, thanks again.
